fact_extractor/install/unpacker.py

```diff
@@ -154,44 +155,6 @@
             # 7z
             'yasm',
         ],
-        'pip3': [
-            'pluginbase',
-            'git+https://github.com/armbues/python-entropy',  # To be checked. Original dependency was deleted.
-            'git+https://github.com/fkie-cad/common_helper_unpacking_classifier.git',
-            'git+https://github.com/fkie-cad/fact_helper_file.git',
-            'git+https://github.com/wummel/patool.git',
-            'archmage',
```

Comment:
?

Reply:
I was wondering about that but could not find any usages. Do you know where this comes from?